jwst/extract_1d/extract_1d_step.py

```diff
@@ -186,6 +187,15 @@ def process(self, input):
         elif isinstance(input_model, ModelContainer):
             self.log.debug('Input is a ModelContainer')
         elif isinstance(input_model, datamodels.MultiSlitModel):
+            #  If input is a 3D rateints (which is unsupported) skip the step
+            try:
+                if len((input_model[0]).shape) == 3:
+                    self.log.warning('ext_1d_step step will be skipped as 3D input is unsupported')
+                    input_model_copy.meta.cal_step.extract_1d = 'SKIPPED'
```

Comment:
Again, don't see the need for a copy here. Just update the meta value in the original input model and return it.

Reply:
fixed